applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -185,6 +224,62 @@ static int channel_index_get(const struct bt_conn *conn, const struct bt_audio_s
 	return -EINVAL;
 }
 
+static void audio_play_pause(bool play)
+{
+	int ret;
+
+	LOG_DBG("Play/pause cb, state: %d", play);
+
+	if (play) {
```

Comment:
These IFs tend to lessen the readability. Suggest that this function only does:
```
static void audio_play_pause(bool play) {
    if(play) {
        audio_play();
    } else {
        audio_pause();
    }

}

```

Reply:
And separate out the rest into other static functions. I see that this was copied from an earlier implementation.

Reply:
Will look into this.